### ISSUE TITLE
Does not support annotationProcessors

### ISSUE THREAD

**Issue by renatoathaydes** [groovy/groovy-eclipse#138]
The Maven Compiler plugin supports declaration of annotation processors like this:

``` xml
<annotationProcessors>
<annotationProcessor>org.checkerframework.checker.nullness.NullnessChecker</annotationProcessor>
</annotationProcessors>
```

When I use this with the Groovy Eclipse plugin, an error occurs:

```
Compilation failure
[ERROR] Failure executing groovy-eclipse compiler:
[ERROR] Unrecognized option : -processor
```

**Comment by eric-milles** [comment]
Batch compiler does have support for this option.  There is a bug in the compiler adapter.

```java
        if (config.getAnnotationProcessors() != null) {
            StringBuilder procArg = new StringBuilder();
            for (String proc : config.getAnnotationProcessors()) {
                if (proc != null && proc.trim().length() > 0) {
                    procArg.append(proc);
                    procArg.append(",");
                }
            }
            if (procArg.length() > 0) {
                procArg.replace(procArg.length() - 1, procArg.length(), "");
                args.put("-processor ", "\"" + procArg.toString() + "\"");
            }
        }
```
Extra space after `-processor` prevents recognition of the compiler argument.

**Comment by eric-milles** [comment]
Fixed compiler adapter (revision 2.9.2-02) available through bintray.

```xml
  <pluginRepositories>
    ...
    <pluginRepository>
      <id>bintray</id>
      <name>Groovy Bintray</name>
      <url>https://dl.bintray.com/groovy/maven</url>
      <releases>
        <updatePolicy>never</updatePolicy>
      </releases>
      <snapshots>
        <enabled>false</enabled>
      </snapshots>
    </pluginRepository>
  </pluginRepositories>
```
